packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ProjectController.java

```diff
@@ -265,6 +272,7 @@ public ResponseEntity<List<Project>> getProjects(
 			}
 		});
 
+		log.info("[getAllProject] End {}", System.currentTimeMillis());
```

Comment:
What is up with all the `log.info` timing stuff? Are they for a different task?

Reply:
Like I said in the description: this is to found out were the issue is with getAllProjects. I will remove them soon.